node/src/chain_spec/turing.rs

```diff
@@ -38,6 +35,8 @@ pub fn turing_development_config() -> ChainSpec {
 	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
 	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
 
+	const DEFAULT_PARA_ID: u32 = 2000;
```

Comment:
seems fine to leave this up top as it is the default and should be used by all by the live

Reply:
done